packages/neuron-ui/src/services/remote/wallets.ts

```diff
@@ -19,6 +19,7 @@ export const generateTx = remoteApi<Controller.GenerateTransactionParams>('gener
 export const generateSendingAllTx = remoteApi<Controller.GenerateSendingAllTransactionParams>('generate-send-all-tx')
 export const generateMnemonic = remoteApi<void>('generate-mnemonic')
 export const validateMnemonic = remoteApi<string>('validate-mnemonic')
+export const isWalletXpub = remoteApi<string, boolean>('is-wallet-xpub')
```

Comment:
Could this be returned with `get-current-wallet`?

Reply:
Sometimes only need to know current wallet is watch-only-wallet

Reply:
Indeed, this property is just used in several cases.

But adding one more API makes the system more complex, meanwhile, the `isWalletXpub` returned along with `get-current-wallet` could be stored in [state](https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/states/init/wallet.ts#L12) and there's no need to fetch `isWalletEpub` asynchronously.

Reply:
Of cause stored it is good idea.